src/components/mortgage-investment-compare/input-card.js

```diff
@@ -123,16 +135,21 @@ const InputCard = props => {
         </FormGroup>
         <hr />
         <FormGroup className='mb-3'>
-          <label>Annual Percentage Rate (APR) & Interest</label>
+          <Tooltip
+            children={<label>Annual Percentage Rate (APR) & Interest</label>}
```

Comment:
sug: How about

```html
<Tooltip>
  <label/>
</Tooltip>
```

to better match HTML structure?

Reply:
yeah that's better, ill make the change